src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -55,29 +53,83 @@ public void sendObjectionSubmittedCustomerEmail (
+    }
+
+    @Override
+    public void sendObjectionSubmittedDissolutionTeamEmail(
+            String requestId,
+            String companyNumber,
+            Objection objection
+    ) throws ServiceException {
 
-        EmailContent emailContent = new EmailContent.Builder()
-                .withOriginatingAppId(originatingAppId)
+        CompanyProfileApi companyProfile = companyProfileService.getCompanyProfile(companyNumber, requestId);
+        String companyName = companyProfile.getCompanyName();
+        Map<String, Object> data = constructEmailDataMap(companyName, companyNumber, objection);
+
+        for (String emailAddress : getDissolutionTeamRecipients(companyProfile.getJurisdiction())) {
+            EmailContent emailContent = constructEmailContent(EmailType.DISSOLUTION_TEAM,
+                    requestId, emailAddress, data);
+            logger.debugContext(requestId, String.format("Calling Kafka client to send dissolution team emailto %s",
+                    emailAddress));
+            kafkaEmailClient.sendEmailToKafka(emailContent);
+            logger.debugContext(requestId, "Successfully called Kafka client");
+        }
+    }
+
+    private EmailContent constructEmailContent(EmailType emailType,
+                                               String requestId,
+                                               String emailAddress,
+                                               Map<String, Object> data) {
+
+        String typeOfEmail = (emailType == EmailType.CUSTOMER)? emailConfig.getSubmittedCustomerEmailType()
+                : emailConfig.getSubmittedDissolutionTeamEmailType();
+
+        return new EmailContent.Builder()
+                .withOriginatingAppId(emailConfig.getOriginatingAppId())
                 .withCreatedAt(dateTimeSupplier.get())
-                .withMessageType(submittedCustomerEmailType)
+                .withMessageType(typeOfEmail)
                 .withMessageId(UUID.randomUUID().toString())
                 .withEmailAddress(emailAddress)
                 .withData(data)
                 .build();
+    }
 
-        logger.debugContext(requestId, "Calling Kafka client to send email");
-        kafkaEmailClient.sendEmailToKafka(emailContent);
-        logger.debugContext(requestId, "Successfully called Kafka client");
+    private Map<String, Object> constructEmailDataMap(String companyName, String companyNumber, Objection objection) {
+        Map<String, Object> data = new HashMap<>();
+
+        data.put("company_name", companyName);
+        data.put("company_number", companyNumber);
+        data.put("objection_id", objection.getId());
+        data.put("reason", objection.getReason());
+        data.put("attachments", objection.getAttachments());
+        return data;
+    }
+
+    protected String[] getDissolutionTeamRecipients(String jurisdiction) {
+        switch(jurisdiction) {
+            case "england":
```

Comment:
Personally i'd prefer not to let england and wales fall through into england-wales and explicitly define their behaviour (even though its the same as england-wales) just to improve code readability

Reply:
all cases now with return statements